pkg/controller/runtimeextractor/controller.go

```
@@ -0,0 +1,169 @@
+}
+
+func (re *runtimeExtractorController) handleVersionUpdate(ctx context.Context) {
+	klog.Info("[RuntimeExtractorController]: Update Deployment")
+	cfg := re.config.Config()
+
+	if cfg.DataReporting.DisableRuntimeExtractor {
+		return
+	}
+	// TODO: version
+	// TODO: handle error?
+	re.updateDeployment(ctx)
```

golangci-lint (2.11.4)

[error] Error return value of `re.updateDeployment` is not checked

(errcheck)